File: repos/Gateway4CSharp/GWDataCenter/MqService/Models/MqRtStateMessage.cs
using System.Collections.Generic;

namespace GWDataCenter.MqService;

public class MqRtStateMessage
{
    public string Time { get; set; }
    public string Flow { get; set; }
    public List<StateItem> StateItems { get; set; } = [];
}
public class StateItem
{
    public int DeviceId { get; set; }
    public string State { get; set; }
}

File: repos/Gateway4Java/gwminidatacenter/src/main/java/gwdatacenter/mqservice/MqRtStateMessage.java
package gwdatacenter.mqservice;

import gwdatacenter.*;
import java.util.*;

public class MqRtStateMessage
{
	private String Time;
	public final String getTime()
	{
		return Time;
	}
	public final void setTime(String value)
	{
		Time = value;
	}
	private String Flow;
	public final String getFlow()
	{
		return Flow;
	}
	public final void setFlow(String value)
	{
		Flow = value;
	}
	private ArrayList<StateItem> StateItems = new ArrayList<>();
	public final ArrayList<StateItem> getStateItems()
	{
		return StateItems;
	}
	public final void setStateItems(ArrayList<StateItem> value)
	{
		StateItems = value;
	}
}